src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/OffenderBooking.java

```diff
@@ -483,6 +483,7 @@ public List<OffenderPropertyContainer> getActivePropertyContainers() {
 
     public List<OffenderProfileDetail> getActiveProfileDetails() {
         return profileDetails.stream()
+            .filter(pd -> Objects.nonNull(pd.getCode()))
```

Comment:
so that this method returns same values, filter out the extra records.
(Since the only client of this method filtered nulls anyway there is no test changes - but feels right not to change this behaviour)